tests/test_bidsInterface.py

```diff
@@ -1,10 +1,16 @@
-from rtCommon.bidsInterface import BidsInterface
+import os
 import pytest
-import numpy
-from tests.backgroundTestServers import BackgroundTestServers
+from rtCommon.bidsArchive import BidsArchive
+from rtCommon.bidsIncremental import BidsIncremental
+from rtCommon.imageHandling import convertDicomImgToNifti, readDicomFromFile
 from rtCommon.clientInterface import ClientInterface
-from rtCommon.projectUtils import npToPy
+from rtCommon.bidsInterface import BidsInterface, tmpDownloadOpenNeuro
+from rtCommon.bidsCommon import getDicomMetadata
+from tests.backgroundTestServers import BackgroundTestServers
+from tests.common import rtCloudPath
 
+test_sampleProjectDicomPath = os.path.join(rtCloudPath,
+    "projects/sample/dicomDir/20190219.0219191_faceMatching.0219191_faceMatching/")
```

Comment:
Would be more cross-platform compatible to separate the words out so that `os.path.join` can handle which separator to use